package.json

```diff
@@ -7,6 +7,7 @@
     "dev": "cross-env NODE_ENV=development webpack-dev-server --inline --progress",
     "start": "serve build -s -c 1",
     "prestart": "npm run build",
+    "preversion": "npm run release",
```

Comment:
If you want the new version number, then this should be `version` instead of `preversion`.

Reply:
Yeah fixing that now.